### PR TITLE
added nominal value for internal energy in XellMixedAir

### PR DESCRIPTION
**Pull request by Mathadon**
see #797

### CODE REVIEW

**Comment by Mathadon** [comment]
this can be merged after #804 
Reference results have been updated. Changes are caused by the new nominal value of U.

**Comment by damienpicard** [comment]
changes are fine but unit tests need to be re-ran.

File: IDEAS/Buildings/Components/ZoneAirModels/WellMixedAir.mo [hunk 1]
```diff
@@ -22,7 +22,8 @@ protected
     allowFlowReversal=allowFlowReversal,
     V=Vtot,
     mSenFac=mSenFac,
-    use_C_flow=true)                           annotation (Placement(
+    use_C_flow=true,
+    dynBal(U(nominal=mSenFac*10*Vtot*1.2*1000)))                           annotation (Placement(
```
Comment (damienpicard) [inline IDEAS/Buildings/Components/ZoneAirModels/WellMixedAir.mo]: Where does this factor 10 come from?

Comment (Mathadon) [reply IDEAS/Buildings/Components/ZoneAirModels/WellMixedAir.mo]: I essentially assume that temperature differences are within a range of 10 K.

**Comment by Mathadon** [comment]
This change causes unexpected result differences. Flow reverses in PPD12, while the nominal value of U should not even be used there since Euler is used? Furthermore, relatively large result differences are obtained for the relative humidity in `ZoneExample`. To be debugged further.

**Pull request merged**